test/mocha/20-helpers.js

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2014-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const server = require('bedrock-server');
+const path = require('path');
+const mockCertBundle = path.join(__dirname, './mock.bundle.crt');
+
+describe('bedrock-server helpers', function() {
+  describe('_parseCerts', function() {
+    it('properly parses a cert bundle', async () => {
+      const caFiles = mockCertBundle;
+      const parsedCerts = server._helpers._parseCerts({caFiles});
+      parsedCerts.should.be.an('array');
+      parsedCerts.length.should.equal(2);
+    });
+  });
+});
```

Comment:
While this test does check the helper ... we also need need to test starting up an actual server using this feature. At a bare minimum, you should confirm you can start a server using this feature manually. It also looks like this `ca` feature may be for trusting client-side certificates, not setting up a bundle for the server as we had previously discussed. That's probably done through the `cert` property -- it's just that it allows cert chain bundles there as well. That doesn't really change anything here, just noting it.